charts/sf-archival/charts/spark-manager/templates/configmap.yaml

```diff
@@ -125,7 +125,7 @@ data:
               "spark.hadoop.fs.azure.account.key.{{ .Values.global.secrets.azure.STORAGE_ACCOUNT_NAME }}.blob.core.windows.net={{ .Values.global.secrets.azure.STORAGE_ACCOUNT_ACCESS_KEY }}",
               "--conf",
               "spark.hadoop.fs.azure=org.apache.hadoop.fs.azure.NativeAzureFileSystem",
-{{- else }}
+{{- else if and (eq .Values.global.secrets.aws.enable true) (eq .Values.global.secrets.aws.use_iam_role false) }}
```

Comment:
No need to mention boolean true or false here. Pls simply this by seeing other places

Reply:
Made changes. Please verify.